README.md

```diff
@@ -6,7 +6,7 @@ The web UI for Estuary Flow.
 
 You'll need to make sure the UI ahas the depencies installed. For the UI this should be straight forward and running `npm install` in the UI root directory.
```

Comment:
There is are typos in the first sentence of this paragraph: _You'll need to make sure the UI **has** the **dependencies** installed._